kernelci/data/kernelci_api.py

```diff
@@ -69,6 +69,12 @@ def get_node(self, node_id):
     def get_node_from_event(self, event):
         return self.get_node(event.data['id'])
 
+    def _put(self, path, data=None):
```

Comment:
Maybe move it next to `_get()` and `_post()` to keep the primitive HTTP wrappers together

Reply:
Okay.